Tests/eppo/RuleEvaluatorTests.swift

```diff
@@ -2,6 +2,8 @@ import XCTest
 
 @testable import eppo_flagging
 
+// todo: add obfuscated tess.
```

Comment:
Do you still want to do this or happy with coverage by the obfuscated client tests?